### PR TITLE
createDirectories without permissions in ExploreRuntimeModule

### PR DESCRIPTION
**Pull request by CuriousVini**
Issue: https://issues.cask.co/browse/CDAP-7829
Build: http://builds.cask.co/browse/CDAP-RUT359-2

### CODE REVIEW

File: cdap-explore/src/main/java/co/cask/cdap/explore/guice/ExploreRuntimeModule.java [hunk 1]
```diff
@@ -157,7 +157,7 @@ private static File createLocalDir(CConfiguration cConf, String dirName) {
       File credentialsDir = new File(localDirStr, dirName);
 
       try {
-        java.nio.file.Files.createDirectories(credentialsDir.toPath(), FileUtils.OWNER_ONLY_RWX);
```
Comment (anwar6953) [inline cdap-explore/src/main/java/co/cask/cdap/explore/guice/ExploreRuntimeModule.java:160]: can we remove FileUtils.OWNER_ONLY_RWX now

**Comment by CuriousVini** [comment]
@anwar6953 addressed comments

**Comment by anwar6953** [comment]
This should be fine, since we use restricted permissions when writing files inside that directory.

**Comment by anwar6953** [comment]
LGTM

**Comment by CuriousVini** [comment]
Build is green. Squashing and merging.

**Pull request merged**